metaflow/plugins/kfp/kfp_cli.py

```diff
@@ -164,6 +165,35 @@ def step_init(obj, run_id, step_name, passed_in_split_indexes, task_id):
     help="Use Argo CLI watch to wait for KFP run to complete.",
     show_default=True,
 )
+@click.option(
+    "--notify",
+    "-notify",
+    "notify",
+    is_flag=True,
+    default=bool(strtobool(from_conf("METAFLOW_NOTIFY", default="False"))),
+    help="Whether to notify upon completion. METAFLOW_NOTIFY_ON_SUCCESS and "
+    "METAFLOW_NOTIFY_ON_ERROR env variables determine whether a notification is sent.",
+    show_default=True,
```

Comment:
Does this mean that if a user triggers the flow with `python myflow.py kfp run --notify` it'll automatically add in the `notify` `ExitHandler` (ie. the user doesn't need to actually attach a decorator)?

Reply:
Yes! And our CICD path can always always compile the Flow with `--yaml-only --notify`.